UI/Web/src/_manga-reader-common.scss

```diff
@@ -7,18 +7,31 @@ img {
   align-items: center;
 
   &.full-width {
-    height: calc(var(--vh)*100);
+    width: 100%;
     display: grid;
+
+    canvas {
```

Comment:
Canvas style should only be in the canvas renderer

Reply:
Resolved with https://github.com/Kareadita/Kavita/pull/1892/commits/3adac186e2c7915565094bdd19320380e8a2aabe